backend/src/models/User.js

```diff
@@ -109,4 +109,8 @@ module.exports = {
     type: 'boolean',
     default: false,
   },
+  language: {
```

Comment:
If `Vuex-i18n` calls it `locale` maybe we want to call it `locale` in the backend, too?

Reply:
ja